docs/src/main/paradox/lagom/serving-grpc.md

```diff
@@ -1,60 +1,64 @@
+Scala
+:   @@snip[helloworld.proto](/lagom-interop-test-scala/src/main/protobuf/helloworld.proto) { #protoSources }
+
+Java
+:   @@snip[helloworld.proto](/lagom-interop-test-java/src/main/protobuf/helloworld.proto) { #protoSources }
 
-You will get an abstract class named @scala[`example.myapp.helloworld.grpc.helloworld.AbstractGreeterServiceRouter`]
-@java[`example.myapp.helloworld.grpc.AbstractGreeterServiceRouter`]Create a concrete subclass implementing this 
-wherever you see fit in your project, let's say `controller.GreeterServiceImpl` like so:
+You will get an abstract class named @scala[`com.lightbend.lagom.scaladsl.grpc.interop.AbstractGreeterServiceRouter`]
+@java[`com.lightbend.lagom.javadsl.grpc.interop.AbstractGreeterServiceRouter`]. Create a concrete subclass implementing this 
+wherever you see fit in your project, let's say @scala[`com.lightbend.lagom.scaladsl.grpc.interop.test.HelloGrpcServiceImpl`]@java[`com.lightbend.lagom.javadsl.grpc.interop.test.api.HelloGrpcServiceImpl`] like so:
 
 Scala
-:   @@snip[GreeterServiceImpl.scala](/play-interop-test-scala/src/main/scala/controllers/GreeterServiceImpl.scala) { #service-impl }
+:   @@snip[HelloGrpcServiceImpl.scala](/lagom-interop-test-scala/src/main/scala/com/lightbend/lagom/scaladsl/grpc/interop/test/HelloGrpcServiceImpl.scala) { #service-impl }
 
 Java
-:   @@snip[GreeterServiceImpl.java](/play-interop-test-java/src/main/java/controllers/GreeterServiceImpl.java) { #service-impl }
+:   @@snip[HelloGrpcServiceImpl.java](/lagom-interop-test-java/src/main/java/com/lightbend/lagom/javadsl/grpc/interop/test/api/HelloGrpcServiceImpl.java) { #service-impl }
 
-And then add the router to your Play `conf/routes` file. Note that the router already knows its own path since it is
-based on the package name and service name of the service and therefore the path `/` is enough to get it to end up in the right place
-(in this example the path will be `/helloworld.GreeterService`).
-It cannot be added at an arbitrary path (if you try to do so an exception will be thrown when the router is started).
 
-```
-->     /   controllers.GreeterServiceImpl
-```
+And then, you need to bind the new additional router in your project.
+
+@@@ div { .group-scala }
+
+In Scala, you use `additionalRouter` method when creating your `lagomServer` instance:
+
+Scala
+:   @@snip[HelloLoader.scala](/lagom-interop-test-scala/src/main/scala/com/lightbend/lagom/scaladsl/grpc/interop/test/HelloLoader.scala) { #service-additional-router }
+
+@@@
+
+@@@ div { .group-java }
+
+In Java, you need to bind the service with the additional router:
+
+Java
+:   @@snip[HelloModule.java](/lagom-interop-test-java/src/main/java/HelloModule.java) { #service-additional-router }
+
+And finally enable this module in your `application.conf` file:
+
+Java
+:   @@snip[application.conf](/lagom-interop-test-java/src/main/resources/application.conf) { #service-module }
+
```

Comment:
when renderred, the Java snippet is not too comprehensive:

```java
protected void configure() {
    bindService(
        HelloService.class, HelloServiceImpl.class, additionalRouter(HelloGrpcServiceImpl.class));
  }
```

I think the following would be more comprehensive:

```java
  protected void configure() {
    bindService(
      HelloService.class, HelloServiceImpl.class, 
      additionalRouter(HelloGrpcServiceImpl.class)
    );
  }
```